projects/ADuCM3029_demo_cn0428_cn0429/src/cli.c

```diff
@@ -0,0 +1,986 @@
+*      the documentation and/or other materials provided with the
+*      distribution.
+*    - Neither the name of Analog Devices, Inc. nor the names of its
+*      contributors may be used to endorse or promote products derived
+*      from this software without specific prior written permission.
+*    - The use of this software may or may not infringe the patent rights
+*      of one or more patent holders.  This license does not release you
+*      from the requirement that you obtain separate licenses from these
+*      patent holders to use this software.
+*    - Use of the software either in source or binary form, must be run
+*      on or directly connected to an Analog Devices Inc. component.
+*
+*THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+*INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT, MERCHANTABILITY AND FITNESS FOR A
+*PARTICULAR PURPOSE ARE DISCLAIMED.
+*
+*IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+*EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, INTELLECTUAL PROPERTY
+*RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+*BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+*STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+*THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************
+**/
+
+/***************************** Include Files **********************************/
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include "cli.h"
+#include "ADuCM3029_demo_cn0428_cn0429.h"
+#include "Ringbuffer.h"
+
+/********************************* Definitions ********************************/
+
+
+
+/************************** Variable Definitions ******************************/
+/* Available commands */
+char *CmdCommands[] = {
+	"",
+	"help",
+	"defaultsensor",
+	"sensorsconnected",
+	"readconfigs",
+
+	"readtemp",
+	"readhum",
+	"setmeastime",
+	"startmeas",
+	"stopmeas",
+	"setrtia",
+	"setrload",
+	"setvbias",
+	"setsensitivity",
+	"runeis",
+	"readeis",
+	"readeisfull",
+	"readrcal",
+	"runpulse",
+	"readpulse",
+	"pulseamplitude",
+	"pulseduration",
+	"readsensors",
+	"stopread",
+	"setupdaterate",
+	""
+};
+
+/* Functions for available commands */
+cmdFunc CmdFun[] = {
+	DoNothing,
+	CmdHelp,
+	SetDefaultSns,
+	SnsConnected,
+	RdCfgs,
+
+	RdTemp,
+	RdHum,
+	CfgMeasTime,
+	StartMeas,
+	StopMeas,
+	CfgRtia,
+	CfgRload,
+	CfgVbias,
+	CfgSens,
+	RunEIS,
+	RdEIS,
+	RdEISfull,
+	RdRcal,
+	RunPulse,
+	ReadPulse,
+	SetPulseAmpl,
+	SetPulseDuration,
+	RdSensors,
+	StopRd,
+	CfgUpdateRate,
+	NULL
+};
+
+
+
+/*Local Variables */
+uint8_t uiDefaultSite = 0;
+uint8_t uiDefaultAddress = 0;
+uint8_t uiPulseAmpl = 0;
+uint8_t uiPulseDur = 0;
+char strCmd[64] = "";
+
+uint8_t		cmdReceived =
```

Comment:
When we right code for drivers, CFTLs, etc, we try to conform to the following guidelines:

https://gitlab.analog.com/Platformation/sw-tools/blob/master/documentation/coding-style.md
https://gitlab.analog.com/Platformation/sw-tools/blob/master/documentation/beautifying-code.md

The coding style applied is configured to force a maximum code length of 80 characters per line.  As mentioned in the documentation above: "The limit on the length of lines is 80 columns and this is a strongly preferred limit."

In order to get Astyle working at its best (as you might see, sometimes it is doing strange formatting), it is recommended to help it by trying follow the rules during the code writing. 

In this situation, I suggest defining the variable on a line, and move the comment above it (or reduce the comment length). Then apply astyle again on this file.

Reply:
Thanks. But I can't access the gitlab documentation. I get a 404 error. Links seem to be broken?

Reply:
I think the access should be resolved now